templates/download/server/choose.html

```diff
@@ -0,0 +1,134 @@
+        </form>
+      </p>
+      <p>
+        <a href="https://wiki.ubuntu.com/{{ releases.lts.slug }}/ReleaseNotes" class="p-link--external">Ubuntu {{ releases.lts.short_version }} LTS release notes</a>
+      </p>
+      <p>
+        <a href="https://www.brighttalk.com/webcast/6793/398353/ubuntu-20-04-lts-whats-new-in-server" class="p-link--external">Watch the webinar &ndash; "Ubuntu 20.04 LTS: What’s new in server?"</a>
+      </p>
+      <h4 class="p-muted-heading">Explore:</h4>
+      <ul class="p-inline-list">
+        <li class="p-inline-list__item">
+          <a href="https://multpass.run" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Server download', 'eventAction' : 'Select option', 'eventLabel' : 'Multipass', 'eventValue' : undefined });">
+            <img src="https://assets.ubuntu.com/v1/4e0ff71e-Multipass+logo_thin_rgb.svg" alt="" width="543" height="104" style="height: 26px; width: auto;" />
+        </li>
+        <li class="p-inline-list__item">
+          <a href="https://maas.io" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Server download', 'eventAction' : 'Select option', 'eventLabel' : 'MAAS', 'eventValue' : undefined });">
+            <img src="https://assets.ubuntu.com/v1/038514e8-MAAS_black-orange_rgb.svg" alt="" width="543" height="104" style="height: 26px; width: auto;" />
+          </a>
+        </li>
+      </ul>
+    </div>
+    <div class="col-4 col-start-large-8 u-hide--small u-align--center u-vertically-center">
+      <div class="u-hide--small u-align--center">
+        {{
+          image(
+            url="https://assets.ubuntu.com/v1/28e0ebb7-Focal-Fossa-gradient-outline.svg",
+            alt="",
+            height="1500",
+            width="1922",
+            hi_def=True,
+            loading="lazy"
+          ) | safe
+        }}
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light u-no-padding--bottom">
+  <div class="row">
+    <div class="col-10">
+      <h2>Alternative downloads</h2>
+    </div>
+  </div>
+  <div class="row u-equal-height">
+    <div class="col-4 p-card">
+      <h3 class="p-card__title">Ubuntu Server for ARM</h3>
+      <p>The next version of Ubuntu Server, to be officially released in October 2020.</p>
+      <p><a href="http://cdimage.ubuntu.com/ubuntu-server/daily-live/current/" class="p-link--external">Get Ubuntu Server 20.10</a></p>
+    </div>
+    <div class="col-4 p-card">
+      <h3 class="p-card__title">Ubuntu Server 18.04 LTS</h3>
+      <p>Ubuntu is available on the IBM POWER platform, bringing the entire Ubuntu and OpenStack ecosystem to IBM POWER.</p>
+      <p><a href="http://cdimage.ubuntu.com/ubuntu/releases/18.04/release/" class="p-link--external">Get Ubuntu Server 18.04 LTS</a></p>
```

Comment:
shouldn't we use the form as above to the thank you page that allows for mirrors?